lib/const.rb

```diff
@@ -75,20 +62,63 @@
   CREW_BUILD = 'x86_64-cros-linux-gnu'
 end
 
-CREW_COMMON_FLAGS = "'-Os -pipe -flto=auto -fuse-ld=gold'"
-CREW_COMMON_FNO_LTO_FLAGS = "'-Os -pipe -fno-lto -fuse-ld=gold'"
+CREW_COMMON_FLAGS = "'-w -Os -pipe -flto=auto -fuse-ld=gold'"
+CREW_COMMON_FNO_LTO_FLAGS = "'-w -Os -pipe -fno-lto -fuse-ld=gold'"
 CREW_FNO_LTO_LDFLAGS = "'-fno-lto'"
 CREW_LDFLAGS = "'-flto=auto'"
 
-CREW_ENV_OPTIONS = "CFLAGS=#{CREW_COMMON_FLAGS} CXXFLAGS=#{CREW_COMMON_FLAGS} FCFLAGS=#{CREW_COMMON_FLAGS} FFLAGS=#{CREW_COMMON_FLAGS} LDFLAGS=#{CREW_LDFLAGS}"
-CREW_OPTIONS = "--prefix=#{CREW_PREFIX} --libdir=#{CREW_LIB_PREFIX} --mandir=#{CREW_MAN_PREFIX} --build=#{CREW_BUILD} --host=#{CREW_TGT} --target=#{CREW_TGT} --program-prefix='' --program-suffix=''"
-CREW_MESON_OPTIONS = "-Dprefix=#{CREW_PREFIX} -Dlibdir=#{CREW_LIB_PREFIX} -Dmandir=#{CREW_MAN_PREFIX} -Dbuildtype=minsize -Db_lto=true -Dcpp_args=#{CREW_COMMON_FLAGS} -Dc_args=#{CREW_COMMON_FLAGS}"
-CREW_MESON_FNO_LTO_OPTIONS = "-Dprefix=#{CREW_PREFIX} -Dlibdir=#{CREW_LIB_PREFIX} -Dmandir=#{CREW_MAN_PREFIX} -Dbuildtype=minsize -Db_lto=false -Dcpp_args=#{CREW_COMMON_FNO_LTO_FLAGS} -Dc_args=#{CREW_COMMON_FNO_LTO_FLAGS}"
+CREW_ENV_OPTIONS = <<~OPT
+  CFLAGS=#{CREW_COMMON_FLAGS} \
+  CXXFLAGS=#{CREW_COMMON_FLAGS} \
+  FCFLAGS=#{CREW_COMMON_FLAGS} \
+  FFLAGS=#{CREW_COMMON_FLAGS} \
+  LDFLAGS=#{CREW_LDFLAGS}
+OPT
```

Comment:
Doesn't this add extra spaces which makes the actual output even longer on a single line?

Reply:
Not with the ~ after OPT. That causes ruby to cut excess indents from the string.

Reply:
@uberhacker Ruby will delete the spaces at start automatically when in `~` mode
@saltedcoffii That's fine as we have space already before the backsplash